src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -22,4 +22,30 @@ public Integer getLength() {
     public String getColor() {
         return color;
     }
+
+    @Override
+    public boolean equals(Object object) {
+        if (object == null) {
+            return false;
+        }
+        if (this == object) {
+            return true;
+        }
+        if (object.getClass().equals(Rectangle.class)) {
+            Rectangle figure = (Rectangle) object;
+            return (width == figure.width || width.equals(figure.width))
```

Comment:
what if width = null, but figure.width is not, then we will have NPE in the right part

Reply:
Solved, i think.  =) 
In this last commit IDE wants to replace it with Objects.equals()...